src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -472,29 +472,29 @@ export class VSCodeNotebookController implements Disposable, IVSCodeNotebookCont
             );
 
             // In development mode, ipywidgets is not under the 'out' folder.
-            scripts.push(
-                Uri.joinPath(
-                    this.context.extensionUri,
-                    'node_modules',
-                    '@vscode',
-                    'jupyter-ipywidgets7',
-                    'dist',
-                    'ipywidgets.js'
-                )
-            );
+            // scripts.push(
```

Comment:
Update build scripts to ensure dev compile and release modes have the same paths, as this is a debt item and we'd like to remove this debt now.